components/authn-service/tokens/pg/adapter.go

```diff
@@ -36,8 +41,33 @@ func (a *adapter) CreateTokenWithValue(ctx context.Context,
 		}
 		id = uid.String()
 	}
+	err := a.validateTokenInputs(ctx, description, []string{}, projects, false)
+	if err != nil {
+		return nil, err
+	}
+
+	return a.insertToken(ctx, id, description, value, active, projects)
+}
+
+func (a *adapter) validateTokenInputs(ctx context.Context,
+	description string, oldProjects, updatedProjects []string, isUpdateRequest bool) error {
+	if emptyOrWhitespaceOnlyRE.MatchString(description) {
+		return status.Error(
+			codes.InvalidArgument,
```

Comment:
Only tokens and teams--the stuff outside authz-service--need the explicit wrapper to add the InvalidArgument. All the stuff inside authz-service gets that added higher up (not in `validate*inputs`).